### PR TITLE
Updating the regex to accept multiple semi colon parsing. Added tests…

### PR DESCRIPTION
**Pull request by RobertVelton**
… to support new functionality.

### CODE REVIEW

File: src/Tokenizer.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function getToken($string)
                     return $matched;
                 }
             ],
-            "/^&[^\s]*;/"              => ['type' => 'html-entity', 'value_fn' => $returnMatches],
+            "/^&[^\s;]*;/"              => ['type' => 'html-entity', 'value_fn' => $returnMatches],
```
Comment (zacharymarshal) [inline src/Tokenizer.php]: I would fix your spacing here 😄

**Review by zacharymarshal — CHANGES_REQUESTED** [review]
Looks good.

File: src/Tokenizer.php (hunk 1 above)
Comment (RobertVelton) [reply src/Tokenizer.php]: This is fixed now! Approves it sir!!@#!@

**Comment by zacharymarshal** [comment]
Haha all good now. Merge it! 📽

**Pull request merged**